### PR TITLE
Add Californium page

### LINKED ISSUE #31: Californium page is missing
# Element Request

## Type of Request

- [x] Addition
- [ ] Edit

## Element Name

Californium

## Description

Californium is a synthetic chemical element; it has symbol Cf and atomic number 98. It was first synthesized in 1950 at Lawrence Berkeley National Laboratory by bombarding curium with alpha particles

### PR DESCRIPTION
**Pull request by javabster**
## Description
This PR adds a page for the element Californium

## Todos
Notable points that this PR has either accomplished or will accomplish.
  - [x] Add element name and atomic number
  - [x] Add description of element
  - [x] Add fun facts for californium

## Questions
none

## Status
- [ ] Ready to go

closes #31

### CODE REVIEW

File: elements/californium.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+layout: page
```
Comment (GirnarJoshi) [inline elements/californium.md:2]: Correct page formatting- good job!

File: elements/californium.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+Atomic Number : 98
+
+Californium was first synthesized in 1950 at Lawrence Berkeley National Laboratory by bombarding curium with alpha particles. 
+
+Fun Facts:
+- It was first synthesized in 1950 at Lawrence Berkeley National Laboratory! 
+- It was named after the state of California and the University of California, Berkeley
+- It has the second-highest atomic mass of all elements that can be seen with the naked eye. 
+- It has 20 known isotopes, with californium-251 being the most stable and having a half-life of 898 years. 
+- It has a melting point of 1,652°F and slowly tarnishes in air at room temperature.
```
Comment (GirnarJoshi) [inline elements/californium.md:16]: What is this °F symbol?? i have never seen it before and feel a shiver down my spine at it's sight. Oh how I miss the vast oceanic land of °C

File: elements/californium.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+layout: page
+title: Californium
+---
+
+Symbol : Cf
+Atomic Number : 98
+
+Californium was first synthesized in 1950 at Lawrence Berkeley National Laboratory by bombarding curium with alpha particles.
```
[the remark this answers is not in the document]
Comment (javabster) [reply elements/californium.md:9]: My sincerest apologies for this gross oversight on my part, I shall address this heinous error at once!

File: elements/californium.md (hunk 2 above)
Comment (javabster) [reply elements/californium.md:16]: I believe °F is an American term, short for "Fuck you metric system!". I'll update to include celsius as well fahrenheit, to be more inclusive

**Review by GirnarJoshi — APPROVED** [review]
Looks Great! A Generation defining Pull Request truly good work :')

**Pull request merged**